File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        int l1 = 1, l2 = 1;
        int result = 1;
        for(int i=2; i<=n; i++) {
            result = l1 + l2;
            l1 = l2;
            l2 = result;
        }
        return result;
    }
}

File: 0070-climbing-stairs/0070-climbing-stairs.go
func climbStairs(n int) int {
	var l1 int = 1
	var l2 int = 1
	var result int = 1
	for i := 2; i <= n; i++ {
		result = l1 + l2
		l1 = l2
		l2 = result
	}
	return result
}